js/main.js

```diff
@@ -60,12 +60,17 @@ document.addEventListener('DOMContentLoaded', function (event) {
   }
 });
 
-/* const $noEntries = document.querySelector('.no-entries')
+const $noEntries = document.querySelector('.no-entries');
 
- function toggleNoEntries() {
- $noEntries.classList.toggle('hide');
+function toggleNoEntries() {
+  if (data.entries.length > 0) {
+    $noEntries.classList.add('hidden');
+  } else {
+    $noEntries.classList.remove('hidden');
+  }
+}
+toggleNoEntries();
```

Comment:
This should be getting called inside your `DOMContentLoaded` callback function & inside your `submit` callback function